$OpenBSD: patch-configure,v 1.19 2011/03/07 19:36:17 ajacoutot Exp $

XXX remove -DGTK_DISABLE_DEPRECATED to fix build with gtk>=2.24

--- configure.orig	Sun Feb  6 05:36:30 2011
+++ configure	Sat Feb 26 21:38:23 2011
@@ -4344,7 +4344,6 @@ fi
   list="-DG_DISABLE_DEPRECATED
 	-DPANGO_DISABLE_DEPRECATED
 	-DGDK_PIXBUF_DISABLE_DEPRECATED
-	-DGTK_DISABLE_DEPRECATED
 	-DEDS_DISABLE_DEPRECATED
 	-DG_DISABLE_SINGLE_INCLUDES
 	-DGTK_DISABLE_SINGLE_INCLUDES
@@ -4405,8 +4404,6 @@ $as_echo "$as_me: WARNING: unsupported compiler flags:
 AM_CPPFLAGS="$WARNING_FLAGS -fno-strict-aliasing"
 
 
-LDFLAGS="$LDFLAGS -Wl,--no-undefined"
-
 # Make sure we can run config.sub.
 $SHELL "$ac_aux_dir/config.sub" sun4 >/dev/null 2>&1 ||
   as_fn_error $? "cannot run $SHELL $ac_aux_dir/config.sub" "$LINENO" 5
@@ -16019,9 +16016,6 @@ LIBS=$ac_check_lib_save_LIBS
 fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_cv_lib_resolv_res_query" >&5
 $as_echo "$ac_cv_lib_resolv_res_query" >&6; }
-if test "x$ac_cv_lib_resolv_res_query" = x""yes; then :
-  LDAP_LIBS="-lresolv"
-fi
 
 		{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for bind in -lsocket" >&5
 $as_echo_n "checking for bind in -lsocket... " >&6; }
@@ -16824,7 +16818,7 @@ $as_echo_n "checking for Kerberos 5... " >&6; }
 		LIBS_save="$LIBS"
 
 		mitlibs="-lkrb5 -lk5crypto -lcom_err -lgssapi_krb5"
-		heimlibs="-lkrb5 -lcrypto -lasn1 -lcom_err -lroken -lgssapi"
+		heimlibs="-lkrb5 -lcrypto -lasn1 -lcom_err -lgssapi"
 		sunlibs="-lkrb5 -lgss"
 		if test "${ac_cv_lib_kerberos5+set}" = set; then :
   $as_echo_n "(cached) " >&6
